spec/models/article_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  it 'will not create without a title' do
+    described_class.create({ body: 'This is a body....' })
+    expect(described_class.count).to eq 0
```

Comment:
Any particular reason for using different type of validation in lines 6 vs. 11/16?

Reply:
not particularly, was just trying out different matchers I saw